molecule/default/converge.yml

```diff
@@ -2,11 +2,70 @@
 - name: Converge
   hosts: all
 
+  vars:
+    dirsrv_suffix: "dc=test,dc=local"
+
+    dirsrv_rootdn: "cn=Directory Manager"
+    dirsrv_rootdn_password: "P455W0RD"
+
+    dirsrv_serverid: "test"
+    dirsrv_listen_host: 0.0.0.0
+
+    dirsrv_tls_enabled: false
+
+    # Interoperability between 389DS 1.3.X and 1.4.X
```

Comment:
Can we split this to have a "default" test that is the plainest default possible, i.e. only mandatory variables are set, and create an additional test, for CentOS 8 only, with the compatibility variables?